### DISCUSSION TITLE
How is routing handled?

### DISCUSSION THREAD

**Discussion by kevincam3** [directus-labs/agency-os#72] [General]
I can see that OS Agency isn't using the pages folder to handle routing. So how is routing being handled? What page/component(s) are being loaded for the home page? How is the routing being handled under the hood?

**Comment by bryantgillespie** [comment]
The home page and the other items in the Directus _pages_ collection are handled using this Nuxt catch-all route.

https://github.com/directus-labs/agency-os/blob/main/pages/%5B...permalink%5D.vue
